usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1739,13 +1739,13 @@
         "federal_account": OrderedDict(
             [
                 ("owning_agency_name", "treasury_account__federal_account__parent_toptier_agency__name"),
-                ("reporting_agency_name", "submission__reporting_agency_name"),
+                ("reporting_agency_name", "reporting_agency_name"),  # Column is appended to in account_download.py
                 ("submission_period", "submission_period"),  # Column is appended to in account_download.py
                 ("federal_account_symbol", "federal_account_symbol"),  # Column is appended to in account_download.py
                 ("federal_account_name", "treasury_account__federal_account__account_title"),
                 ("agency_identifier_name", "agency_identifier_name"),  # Column is appended to in account_download.py
-                ("budget_function", "treasury_account__budget_function_title"),
-                ("budget_subfunction", "treasury_account__budget_subfunction_title"),
+                ("budget_function", "budget_function"),  # Column is appended to in account_download.py
+                ("budget_subfunction", "budget_subfunction"),  # Column is appended to in account_download.py
```

Comment:
I see changes made to these columns in account_download.py, but they look like exclusively syntax changes. Were these values just being set then overwritten before?

Reply:
Me no understand question.  Previously on _This Block of Code_, the budget function was just being inserted into the file "as is" which caused Federal Accounts to be repeated.  Now we're concatenating the budget function info using StringAgg so that multiple budget functions for a Federal Account appear in the same cell.

Reply:
Where does this StringAgg occur? I don't see any new functions being added, hence my confusion.

Reply:
In account_download.py line 191 (scroll up a little).

Reply:
Ohh, wow I can't read